content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Channel Parameters
+section: realtime
+index: 1
+jump_to:
+  Help with:
+    - Overview#overview
+    - Supported channel parameters#supported-parameters
+    - Using channel parameters#using-parameters
+    - Using channel parameters with non-Ably transports#using-parameters-with-other-transports
+    - Next steps#next-steps
+---
+
+h2(#overview). Overview
+
+Channel parameters allow a client to set the properties of a channel. A set of channel parameters is a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
+
+The methods provided for specifying channel parameters are outlined below.
+
+h2(#supported-parameters). Supported channel parameters
+
+- rewind := Used to request that an attachment start from a given number of messages or point in time in the past. See "rewind":/realtime/channels/channel-parameters/rewind for more information.
+- delta := Used to request that data payloads should be sent as deltas to the previous payload. See "deltas":/realtime/channels/channel-parameters/deltas for more information.
+
+h2(#using-parameters). Using channel parameters
+
+You can specify channel parameters in the @ChannelOptions@ when obtaining a @Channel@. A collection of channel parameters is expressed as a map of string key/value pairs. The @ChannelOptions@ associated with a channel may also be updated by calling "setOptions":/realtime/channels/overview#modifying-options. The parameters associated with a channel take effect when the channel is first attached; if the parameters are subsequently modified via a call to @setOptions@, then that call triggers an attach operation that applies the updated parameters, if successful.
+
+h3. Example
+
+For example, to specify the @rewind@ channel param with the value @"1"@:
+
+```[javascript]
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channelOpts = {params: {rewind: '1'}}
+  const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+To modify the @rewind@ channel param with the value @"15s"@:
+
+```[javascript]
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channelOpts = {params: {rewind: '15s'}}
+  channel.setOptions(channelOpts, (err) => {
+    if(!err) {
+      console.log('channel params updated')
+    }
+  });
+```
+
+h2(#using-parameters-with-other-transports). Using parameters with non-Ably transports
+
+It is possible to interact with Ably channels using transports that do not involve using an Ably library; for example using SSE without any library, or using a supported non-Ably protocol such as MQTT. In these cases, it is also necessary to use a qualified channel name.
+
+A set of params is expressed by including a query string, using standard URL query syntax and encoding, within the qualifier part of a channel name. The qualifier part is in square brackets at the start of the channel name.
```

Comment:
It looks like you've gone for parameters not params, yet params is used here.

Reply:
good spot, thanks. addressed in the update.